src/interfaces/wasi/wit_implementation.rs

```diff
@@ -5,6 +5,16 @@ use super::wit_generation;
 cosmian_wit_bindgen_rust::export!("abe.wit");
 struct Abe;
 
+fn attributes_to_wit_attributes(attributes: Vec<Attribute>) -> Vec<wit_generation::Attribute> {
```

Comment:
maybe we can implement a `From` conversion, what do you think?

Maybe it's not very useful in this case of interface though

Reply:
yes